apps/docs/src/layouts/MobileViewLayout.astro

```diff
@@ -0,0 +1,34 @@
+---
+import HeadCommon from "../components/HeadCommon.astro";
+import HeadSEO from "../components/HeadSEO.astro";
+import * as CONFIG from "../config";
+import type { MarkdownHeading } from "astro";
+import { isDarkTheme } from "../stores";
+
+type Props = {
+  frontmatter: CONFIG.Frontmatter;
+  headings: MarkdownHeading[];
+};
+
+const { frontmatter } = Astro.props as Props;
+const canonicalURL = new URL(Astro.url.pathname, Astro.site);
+const darkTheme = isDarkTheme.get();
```

Comment:
do those nanostores work okay with this code? I mean that part of the frontmatter is executed on buildtime. Haven't worked with it so just asking 🤔

Reply:
Nope, also redundant piece of code here 🤡 Consider it an unwanted leftover from trying to solve a problem 😅